src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/README.md

```diff
@@ -46,13 +47,23 @@ Follow the instructions at [SpectatorView.Setup](../../../../../../doc/Spectator
 
 1. Open the `SpectatorViewCompositor` scene in your application's Unity project (with your build platform set to Universal Windows Platform).
 2. Open the Compositor Window from the Spectator View -> Compositor menu.
-3. Press Play to run the scene.
+3. Select your desired Video Source from the dropdown list 
+4. If the selected Video Source enables occlusion, select desired occlusion mode
+
+>Note: "RawDepthCamera" occludes utilizing only the depth image provided by the camera. "BodyTracking" combines the depth image and body mask provided by Azure Body Tracking SDK to occlude only recognized people
+
+6. Select your desired preview display
+>Note: Occlusion masks are not included in "intermediate textures" preview display type
```

Comment:
>Occlusion [](start = 7, length = 9)

I would clarify this to say "Depth occlusion masks" since there is a holograph opacity inclusion mask included in the intermediate textures display. Possibly we should clarify other usages of "occlusion" as "depth occlusion" or "depth-based occlusion" here too?